src/2dplane/ObstacleGrid.cpp

```diff
@@ -24,6 +28,38 @@ Vector2i ObstacleGrid::gridSquareForLocation(const Vector2f &loc) const {
                     loc.y() / height() * discretizedHeight());
 }
 
+float ObstacleGrid::nearestObstacleDist(const Vector2f &state, float maxDist) const {
+    //x and y are the indices of the cell that state is located in
+    float x = (state.x() / (_width / _discretizedWidth));
+    float y = (state.y() / (_height / _discretizedHeight));
+    int xSearchRad = maxDist * _discretizedWidth / (_width * SQRT2);
```

Comment:
I'm not sure if you need the `SQRT2` here...  Aren't you just trying to convert `maxDist` to discrete coordinates?